samples/samples-python/AddProductParams/__init__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import azure.functions as funcs
```

Comment:
Was this intended to change this to `funcs`?

Reply:
Oh no that was an accident, thanks!